docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,68 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
```

Comment:
Hmm, although I adapted the suggested rule description from “enforce consistent linebreak style for operators” for operator-linebreak, it eventually came to my mind that “enforce consistent linebreak style” for linebreak-style refers to the end-of-line itself `"\n"` versus `"\r\n"`.

So, is “linebreak style” ambiguous in a helpful or harmful way?